### PR TITLE
Added toast and modal component

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. Added modal component.
2. Added toast component

### CODE REVIEW

File: Components/toast.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.toast{
+    display: flex;
+    flex-direction: column;
+    max-width: 250px;
```
Comment (xii-vi) [inline Components/toast.css]: _use relative unit_

File: Components/alert.css [hunk 2]
```diff
@@ -44,7 +44,7 @@
   width: 1.3rem;
   height: 1.3rem;
   padding: 0.5rem 0.5rem;
-  color: var(--dark-secondary-text-color);
+  /* color: var(--dark-secondary-text-color); */
```
Comment (bishnoimukesh) [inline Components/alert.css]: suggested removing comment

**Review by xii-vi — APPROVED** [review]
_Go through the comment once, before merging_

**Review by bishnoimukesh — APPROVED** [review]
suggested changes rest looks good to go

**Pull request merged**